pom.xml

```diff
@@ -1748,6 +1748,7 @@ under the License.
 										<exclude>log4j2.properties</exclude>
 										<exclude>log4j-test.properties</exclude>
 										<exclude>log4j2-test.properties</exclude>
+										<exclude>META-INF/maven/**</exclude>
```

Comment:
TODO: double-check that this also works properly for multi-release jars (e.g., flink-dist)